includes/phing/build/main/subsite.xml

```diff
@@ -80,7 +80,11 @@
         <echo msg="Copy custom resources." />
 
         <copy todir="${build.${build-type}.dir}">
-            <fileset dir="${build.tmp.dir}" expandsymboliclinks="true" />
+            <fileset dir="${build.tmp.dir}" expandsymboliclinks="true">
```

Comment:
I would use the defaultsexclude property: https://www.phing.info/docs/guide/trunk/FileSet.html
But I see it's set to true by default. So actually it was probably already skipping all those files.